reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -87,10 +87,10 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
 
         particles_f_1 = primal_to_fourier_2D(
             torch.from_numpy(particles_1.reshape((1, 1, n_pix, n_pix)))
-        )
+        ).numpy()
```

Comment:
remember to reshape back to what is assumed.

the batch size will be`len(particles_1)`, but here you've written `(1, 1, n_pix, n_pix)` instead of `(n_batch, 1, n_pix, n_pix)`

Reply:
In case it comes up. reshape in numpy is `np.ndarray.reshape((*,*,...))` and in torch it's `torch.tensor.reshape(*,*,...)`. Note the difference in brackets.

Reply:
good catch, thank you